user-interface/src/lib/type-declarations/api.d.ts

```diff
@@ -1,11 +1,12 @@
-export interface SimpleResponseData {
+export interface SimpleResponseData<T = object> {
```

Comment:
Should these interface be something that belongs in the common module?

Reply:
Yes, we should unify these between the API and UI via common. I just was not prepared to do that big refactor with this ticket since it was so big already.

Reply:
Defer to another ticket.